lib/markdown2.py

```diff
@@ -2037,15 +2037,36 @@ def _add_footnotes(self, text):
                 '<hr' + self.empty_element_suffix,
                 '<ol>',
             ]
+
+            try:
+                footnote_title = self.footnote_title
+            except Exception:
+                footnote_title = 'title="Jump back to footnote %d in the text.">'
+
+            try:
+                footnote_link_text = self.footnote_link_text
+            except Exception:
+                footnote_link_text = '&#8617;</a>'
+
             for i, id in enumerate(self.footnote_ids):
                 if i != 0:
                     footer.append('')
                 footer.append('<li id="fn-%s">' % id)
                 footer.append(self._run_block_gamut(self.footnotes[id]))
-                backlink = ('<a href="#fnref-%s" '
-                    'class="footnoteBackLink" '
-                    'title="Jump back to footnote %d in the text.">'
-                    '&#8617;</a>' % (id, i+1))
+                try:
+                    backlink = ('<a href="#fnref-%s" ' +
+                            'class="footnoteBackLink" ' +
+                            footnote_title +
+                            footnote_link_text) % (id, i+1)
+                except Exception as E:
```

Comment:
What Exception are you expecting to see here? Can we be specific about what is caught?